src/components/data-table/types.d.ts

```diff
@@ -0,0 +1,47 @@
+import { ReactNode } from "react";
+
+export type Order = "asc" | "desc";
+
+export interface TableObject {
+    id: string;
+    [key: string]: any;
+}
+
+export interface TableColumn {
+    name: string;
+    text: string;
+    sortable: boolean;
```

Comment:
`sortable? boolean;` (default true?). And implement it, I think now they are always sortable.